diff --git a/pathologic.module b/pathologic.module
index 8f77b03..54c88a0 100644
--- a/pathologic.module
+++ b/pathologic.module
@@ -165,6 +165,9 @@ function _pathologic_replace($matches) {
     )
     // Bail out if it looks like there's only a fragment part.
     || (isset($parts['fragment']) && count($parts) === 1)
+    // Bail out if it contains a Tour, ie Tour module is used and there is a link to autostart a tour on a page, eg
+    // /node/2?=tour=1
+    || (isset($parts['query']) && strpos($parts['query'], '=tour=') !== false)
   ) {
     // Give up by "replacing" the original with the same.
     return $matches[0];
